From: George Wilkie <george.wilkie@intl.att.com>
Date: Tue, 9 Jul 2019 14:52:51 +0100
Subject: Makefile changes to get it building against libre

Remove use of rew - the re work-in-progress library.
The PCP from there has been patched into libre proper.
Add -Werror compile flag
---
 Makefile         | 6 ++----
 include/repcpd.h | 2 +-
 2 files changed, 3 insertions(+), 5 deletions(-)

diff --git a/Makefile b/Makefile
index f6f0291..6b3bf59 100644
--- a/Makefile
+++ b/Makefile
@@ -53,15 +53,13 @@ LIBDIR  := $(PREFIX)/lib
 endif
 MOD_PATH:= $(LIBDIR)/$(PROJECT)/modules
 CFLAGS	+= -I$(LIBRE_INC) -Iinclude
-CFLAGS  += -I$(SYSROOT)/local/include/rew
+CFLAGS  += -Werror
 BIN	:= $(PROJECT)$(BIN_SUFFIX)
 MOD_BINS:= $(patsubst %,%.so,$(MODULES))
 APP_MK	:= src/srcs.mk
 MOD_MK	:= $(patsubst %,modules/%/module.mk,$(MODULES))
 MOD_BLD	:= $(patsubst %,$(BUILD)/modules/%,$(MODULES))
 
-LIBS	+= -lrew
-
 include $(APP_MK)
 include $(MOD_MK)
 
@@ -77,7 +75,7 @@ $(BIN): $(OBJS)
 ifneq ($(GPROF),)
 	@$(LD) $(LFLAGS) $(APP_LFLAGS) $^ ../re/libre.a $(LIBS) -o $@
 else
-	@$(LD) $(LFLAGS) $(APP_LFLAGS) $^ -L$(LIBRE_SO) -lre $(LIBS) -o $@
+	@$(LD) $(LFLAGS) $(APP_LFLAGS) $^ -lre $(LIBS) -o $@
 endif
 
 $(BUILD)/%.o: %.c $(BUILD) Makefile $(APP_MK)
diff --git a/include/repcpd.h b/include/repcpd.h
index 3c30dc7..ea0c207 100644
--- a/include/repcpd.h
+++ b/include/repcpd.h
@@ -5,7 +5,7 @@
  */
 
 
-#include <rew.h>
+#include <re_pcp.h>
 
 
 /*
